Analysis/Tasks/PWGCF/FemtoDream/femtoDreamProducerTask.cxx

```diff
@@ -149,6 +215,36 @@ struct femtoDreamProducerTask {
         }
       }
     }
+
+    for (auto& v0 : fullV0s) {
+      auto postrack = v0.posTrack_as<aod::FilteredFullTracks>();
+      auto negtrack = v0.negTrack_as<aod::FilteredFullTracks>(); ///\tocheck funnily enough if we apply the filter the sign of Pos and Neg track is always negative
+      if (!v0Cuts.isSelectedMinimal(col, v0, postrack, negtrack)) {
+        continue;
+      }
+      v0Cuts.fillQA(col, v0); ///\todo fill QA also for daughters
+      auto cutContainerV0 = v0Cuts.getCutContainer(col, v0, postrack, negtrack);
+      if ((cutContainerV0.at(0) > 0) && (cutContainerV0.at(1) > 0) && (cutContainerV0.at(2) > 0)) {
+        int postrackID = v0.posTrackId();
+        int rowInPrimaryTrackTablePos = -1;
+        rowInPrimaryTrackTablePos = getRowDaughters(postrackID, tmpIDtrack);
+        childIDs[0] = rowInPrimaryTrackTablePos;
+        childIDs[1] = 0;
+        ROOT::Math::PxPyPzMVector postrackVec(v0.pxpos(), v0.pypos(), v0.pzpos(), 0.);
+        ROOT::Math::PxPyPzMVector negtrackVec(v0.pxneg(), v0.pyneg(), v0.pzneg(), 0.);
+        outputTracks(outputCollision.lastIndex(), postrackVec.Pt(), postrackVec.Eta(), postrackVec.Phi(), aod::femtodreamparticle::ParticleType::kV0Child, cutContainerV0.at(1), 0., childIDs);
```

Comment:
Here it would be better to avoid the usage of `ROOT::Math::PxPyPzMVector` in favor of dynamic columns of the v0. I see you need Pt, Eta and Phi, and for Pt and Eta we already have dynamic columns - we only miss Eta. Let me see if we can quickly add that to the strangeness table!

Reply:
@ddobrigk meant that we are missing Phi. He will add this today. Could you already adjust for the others?

Reply:
@jgrosseo actually I was checking an older version of the strangeness tables and it turns out we already added Phi when we added the armenteros-podolanski variables, so in fact all variables used in these posTrackVec/negTrackVec can be safely replaced by calls to dynamic columns right away! The relevant dynamic columns are: [pt](https://github.com/AliceO2Group/AliceO2/blob/072ede13804921da5edb22e24ee310ea87b4287b/Analysis/DataModel/include/AnalysisDataModel/StrangenessTables.h#L47), [eta](https://github.com/AliceO2Group/AliceO2/blob/072ede13804921da5edb22e24ee310ea87b4287b/Analysis/DataModel/include/AnalysisDataModel/StrangenessTables.h#L102) and [phi](https://github.com/AliceO2Group/AliceO2/blob/072ede13804921da5edb22e24ee310ea87b4287b/Analysis/DataModel/include/AnalysisDataModel/StrangenessTables.h#L104)

Reply:
Hi @ddobrigk  and @jgrosseo .
Thanks for the feedback. 
I looked at the columns you refer but those are the pT, eta and phi of the V0. For our analyses we really need to store the pt, eta and phi of the daughters singularly in order to then evaluate the IM when we build the pairs to be used for the correlations.

Reply:
Ah, you're right, of course. In that case, we could easily add dynamic columns to do the calculations for the daughters too. Let me take a look at that and get back to you! 

(Sorry for the insistence. The point is really the desire to keep specific ROOT calls out of the analyses)

Reply:
Aren't the daughters normal tracks and therefore the standard dynamic columns can be used? What am I missing?

Reply:
we need the refitted momenta of the tracks, since we want to compute the inv. mass
and these are properties of the V0 (see e.g. v0.pxpos())

Reply:
but what is the problem with these specific root classes?

Reply:
Hi @ddobrigk, any news on this, or can we for now go ahead?

Reply:
@ddobrigk can you add those dynamic columns? But we can go ahead with this PR anyway...

Reply:
after the codechecker is fixed, of course ;-)

Reply:
I am currently having a look at the codechecker.
Get back to you asap.

Reply:
@sawenzel @jgrosseo I am having trouble understanding the copyright issue.
In all the mentioned files this is the copyright statement (I copied it from the documentation page):
// Copyright CERN and copyright holders of ALICE O2. This software is
// distributed under the terms of the GNU General Public License v3 (GPL
// Version 3), copied verbatim in the file "COPYING".
//
// See http://alice-o2.web.cern.ch/license for full licensing information.
//
// In applying this license CERN does not waive the privileges and immunities
// granted to it by virtue of its status as an Intergovernmental Organization
// or submit itself to any jurisdiction.

Is there something wrong in it?

Reply:
It was changed recently slightly and is probably outdated on the documentation page. Copy it from a file in O2, then you are safe.
@pbuehler could you update the documentation?

Reply:
Perfect thanks!

Reply:
@jgrosseo I updated the copyright and added braces, now the codechecker should work fine.
Sorry did not realize that it has changed.
I would say that once the columns  are implemented (thanks btw to @ddobrigk) I update the commit and we can eventually merge.

Reply:
@sartozza Once the functions from @ddobrigk are there you can update.